eng/Versions.props

```diff
@@ -102,7 +102,7 @@
     <MicrosoftVisualStudioEditorPackageVersion>16.8.272</MicrosoftVisualStudioEditorPackageVersion>
     <MicrosoftVisualStudioLanguagePackageVersion>16.8.272</MicrosoftVisualStudioLanguagePackageVersion>
     <MicrosoftVisualStudioLanguageIntellisensePackageVersion>16.8.272</MicrosoftVisualStudioLanguageIntellisensePackageVersion>
-    <MicrosoftVisualStudioLanguageServerClientImplementationPackageVersion>16.9.121</MicrosoftVisualStudioLanguageServerClientImplementationPackageVersion>
+    <MicrosoftVisualStudioLanguageServerClientImplementationPackageVersion>16.9.150</MicrosoftVisualStudioLanguageServerClientImplementationPackageVersion>
```

Comment:
is this the same package version you use for the LSP protocol packages?  Name sounds like just LSP client implementation so was wondering (as the protocol packages need to be updated)

Reply:
Yes, from my understanding the two packages are co-ordinated. When I [validated earlier](https://github.com/dotnet/aspnetcore-tooling/pull/2796), I used the same version as https://github.com/dotnet/roslyn/commit/381836c779c01750f7cf0393c8facc072cdcf020.